### PR TITLE
Add additional sometimes hints

### PR DESCRIPTION
**Pull request by cjs8487**
Adds the following sometimes hints
- Great Tree
- SV Dig Spot
- Turf Slide
- Slope Boulder Dig Spot (2x20)
- ET West Wing
- Fire Node Shortcut
- Lightning Node Ending
- Floria Chest
- AC East Wing
- SSH Brig Key

### CODE REVIEW

File: checks.yaml [hunk 1]
```diff
@@ -650,10 +650,14 @@ Faron Woods - Chest inside Great Tree:
   type: faron, miscellaneous
   Paths:
     - stage/F100_1/r0/l0/TBox/84
+  hint: sometimes
+  text: the <r<hollow>> tree holds
```
Comment (lepelog) [inline checks.yaml:654]: The coloring pattern seems inconsistent to me, sometimes it spans everything but the verb at the end, sometimes it just highlights a seemingly unrelated word.

I prefer highlighting all the important parts, so `hollow tree` in this case.

Also imo, we can just use `great tree` here.

Unless I'm the only one complaining, we could first get the new sometimes hints in without cryptic messages and add them later.

File: checks.yaml [hunk 2]
```diff
@@ -1143,6 +1161,8 @@ Lanayru Mining Facility - Goddess Harp:
 Ancient Cistern - Chest in East Part:
   original item: AC Small Key
   type: faron, dungeon
+  hint: sometimes
+  text: the <r<lone west chest in the Ancient Cistern>> reveals
```
Comment (lepelog) [inline checks.yaml:1165]: > Chest in East Part
> lone west chest

:thinking:

**Review by lepelog — CHANGES_REQUESTED** [review]
mostly agree with the added hints, I've voiced my concerns about a few locations in discord already so I won't repeat it here

File: checks.yaml (hunk 2 above)
Comment (YourAverageLink) [reply checks.yaml:1165]: This is a bit of a nitpick, but i disagree with the use of "reveals" here too

**Comment by YourAverageLink** [comment]
The lightning node sometimes hint is missing a containing verb (like holds, contains, etc.) after its description

**Review by lepelog — APPROVED** [review]

**Pull request merged**